KPI/dashboards/mobile_kpis.dashboard.lookml

```diff
@@ -0,0 +1,1256 @@
+            id: mobile_usage_2021.delta_from_target_count, name: Delta From Target
+              Count}], showLabels: true, showValues: true, unpinAxis: false, tickDensity: default,
+        tickDensityCustom: 5, type: linear}, {label: !!null '', orientation: right,
+        series: [{axisId: mobile_usage_2021.delta_from_forecast_count, id: mobile_usage_2021.delta_from_forecast_count,
+            name: Delta From Forecast Count}], showLabels: true, showValues: true,
+        unpinAxis: false, tickDensity: default, tickDensityCustom: 5, type: linear}]
+    series_types: {}
+    series_labels:
+      mobile_usage_2021.delta_from_target_count: Difference From Target
+      mobile_usage_2021.delta_from_forecast_count: Difference From Forecast
+    reference_lines: [{reference_type: line, range_start: max, range_end: min, margin_top: deviation,
+        margin_value: mean, margin_bottom: deviation, label_position: center, color: "#000000",
+        line_value: '0', label: At Target (Left) / Forecast (Right)}]
+    defaults_version: 1
+    listen:
+      Date: mobile_usage_2021.date
+    row: 63
+    col: 0
+    width: 11
+    height: 9
+  - title: Cumulative Difference vs Target / Forecast
+    name: Cumulative Difference vs Target / Forecast (4)
+    model: kpi
+    explore: mobile_usage_2021
+    type: looker_line
+    fields: [mobile_usage_2021.date, mobile_usage_2021.delta_from_target_count, mobile_usage_2021.delta_from_forecast_count]
+    fill_fields: [mobile_usage_2021.date]
+    filters:
+      mobile_usage_2021.app_name: '"firefox_ios"'
+    sorts: [mobile_usage_2021.date desc]
+    limit: 500
+    x_axis_gridlines: false
+    y_axis_gridlines: true
+    show_view_names: false
+    show_y_axis_labels: true
+    show_y_axis_ticks: true
+    y_axis_tick_density: default
+    y_axis_tick_density_custom: 5
+    show_x_axis_label: true
+    show_x_axis_ticks: true
+    y_axis_scale_mode: linear
+    x_axis_reversed: false
+    y_axis_reversed: false
+    plot_size_by_field: false
+    trellis: ''
+    stacking: ''
+    limit_displayed_rows: false
+    legend_position: center
+    point_style: none
+    show_value_labels: false
+    label_density: 25
+    x_axis_scale: auto
+    y_axis_combined: true
+    show_null_points: true
+    interpolation: linear
+    y_axes: [{label: '', orientation: left, series: [{axisId: mobile_usage_2021.delta_from_target_count,
+            id: mobile_usage_2021.delta_from_target_count, name: Delta From Target
+              Count}], showLabels: true, showValues: true, unpinAxis: false, tickDensity: default,
+        tickDensityCustom: 5, type: linear}, {label: !!null '', orientation: right,
+        series: [{axisId: mobile_usage_2021.delta_from_forecast_count, id: mobile_usage_2021.delta_from_forecast_count,
+            name: Delta From Forecast Count}], showLabels: true, showValues: true,
+        unpinAxis: false, tickDensity: default, tickDensityCustom: 5, type: linear}]
```

Comment:
Just want to see what you think about pinning both to the same axis. The plots may be a bit more understandable that way, but perhaps they didn't look right when you tried it.